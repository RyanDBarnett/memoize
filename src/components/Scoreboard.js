import React from 'react';
import '../styles/_Scoreboard.scss';

const Header = ({numCards,numCorrect,numIncorrect}) => {
  return (
    <section className="Scoreboard">
      <h2>Cards Left: {numCards}</h2>
      <h2>Correct: {numCorrect}</h2>
      <h2>Incorrect: {numIncorrect}</h2>
    </section>
  )
}

export default Header;