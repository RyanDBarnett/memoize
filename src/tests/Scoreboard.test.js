import React from 'react';
import Scoreboard from '../components/Scoreboard';
import { shallow } from "enzyme";

describe('Scoreboard', () => {
  it('should match snapshot', () => {
    let wrapper = shallow(
      <Scoreboard
        numCards={25}
        numCorrect={3}
        numIncorrect={3}
      />
    )
    expect(wrapper).toMatchSnapshot();
  }); 
});