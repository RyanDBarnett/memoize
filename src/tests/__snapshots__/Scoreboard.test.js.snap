// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scoreboard should match snapshot 1`] = `
<section
  className="Scoreboard"
>
  <h2>
    Cards Left: 
    25
  </h2>
  <h2>
    Correct: 
    3
  </h2>
  <h2>
    Incorrect: 
    3
  </h2>
</section>
`;
